R/expect_lint.R

```diff
@@ -23,6 +24,7 @@
 #' @examples
 #' # no expected lint
 #' expect_lint("a", NULL, trailing_blank_lines_linter())
```

Comment:
Maybe remove this undesirable alternative?

Reply:
Makes sense, I will also remove it from the tests.

Reply:
And added this to the vignette.